docs/release-notes/RELEASE-NOTES-2.0.0-rc.1.md

```diff
@@ -1,16 +1,18 @@
 # About this release
 
-[This][this-release] is a maintenance release.
+[This][this-release] is the first Minerva release candidate.
 It:
 * Does all the things mentioned temporarily in files [/docs/release-notes/next/PT-*.md](/docs/release-notes/next/).
+* Does all the things mentioned temporarily in files [/docs/release-notes/next-minerva/PT-*.md](/docs/release-notes/next-minerva/).
 
 TODO: When preparing the release, concatenate all `/docs/release-notes/next/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
+TODO: When preparing the release, concatenate all `/docs/release-notes/next-minerva/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
 
-[this-release]: https://github.com/aeternity/aeternity/releases/tag/v1.5.0
+[this-release]: https://github.com/aeternity/epoch/releases/tag/v2.0.0-rc.1
```

Comment:
old repo ?

Reply:
🤦‍♂️

Reply:
Addressed in https://github.com/aeternity/aeternity/pull/2126/commits/304ae47976d6559b5178da00511755659e5e16f2
Thanks.